@SuperSandro2000 this the tail for paperless-ngx against master:

```
      > FAILED src/documents/tests/test_management_consumer.py::TestConsumer::test_consume_file - AssertionError: Expected 'delay' to have been called once. Called 0 times.
       > FAILED src/documents/tests/test_management_consumer.py::TestConsumer::test_mac_write - AssertionError: 2 != 0
       > FAILED src/documents/tests/test_management_consumer.py::TestConsumer::test_slow_write_and_move - AssertionError: Expected 'delay' to have been called once. Called 0 times.
       > FAILED src/documents/tests/test_management_consumer.py::TestConsumer::test_slow_write_incomplete - AssertionError: Expected 'delay' to have been called once. Called 0 times.
       > FAILED src/documents/tests/test_management_consumer.py::TestConsumer::test_slow_write_pdf - AssertionError: Expected 'delay' to have been called once. Called 0 times.
       > FAILED src/documents/tests/test_management_consumer.py::TestConsumerRecursive::test_consume_file - AssertionError: Expected 'delay' to have been called once. Called 0 times.
       > FAILED src/documents/tests/test_management_consumer.py::TestConsumerRecursive::test_mac_write - AssertionError: 2 != 0
       > FAILED src/documents/tests/test_management_consumer.py::TestConsumerRecursive::test_slow_write_and_move - AssertionError: Expected 'delay' to have been called once. Called 0 times.
       > FAILED src/documents/tests/test_management_consumer.py::TestConsumerRecursive::test_slow_write_incomplete - AssertionError: Expected 'delay' to have been called once. Called 0 times.
       > FAILED src/documents/tests/test_management_consumer.py::TestConsumerRecursive::test_slow_write_pdf - AssertionError: Expected 'delay' to have been called once. Called 0 times.
       > FAILED src/documents/tests/test_management_consumer.py::TestConsumerTags::test_consume_file_with_path_tags - AssertionError: Expected 'delay' to have been called once. Called 0 times.
       > 11 failed, 1030 passed, 14 skipped, 5 deselected, 95 warnings in 618.42s (0:10:18)
```

Since paperless-ngx is fine on current master without the HDF5 update, I assume that this may be caused by the update (although it is not obvious). However, I leave the fixing of dependent packages to the respective maintainers.

--- a/pkgs/by-name/pd/pdi/package.nix
+++ b/pkgs/by-name/pd/pdi/package.nix
@@ -45,7 +45,7 @@ stdenv.mkDerivation rec {
   # Patch it to make it match a 4 number-long version.
   postPatch = ''
     substituteInPlace plugins/decl_hdf5/cmake/FindHDF5.cmake \
-      --replace-fail '"H5_VERSION[ \t]+\"([0-9]+\\.[0-9]+\\.[0-9]+)' '"H5_VERSION[ \t]+\"([0-9]+\\.[0-9]+\\.[0-9]+\\.[0-9]+)'
+      --replace-fail '"H5_VERSION[ \t]+\"([0-9]+\\.[0-9]+\\.[0-9]+)' '"H5_VERSION[ \t]+\"([0-9]+\\.[0-9]+\\.[0-9]+(\\.[0-9]+)*)'
   '';
 
   nativeBuildInputs = [

